db/schema.rb

```diff
@@ -10,9 +10,23 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 0) do
+ActiveRecord::Schema.define(version: 2021_10_25_043839) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
+  enable_extension "pgcrypto"
+
+  create_table "users", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
+    t.string "name"
+    t.string "email", default: "", null: false
+    t.string "encrypted_password", default: "", null: false
```

Comment:
question: Assuming that devise does the encryption, how does it know to encrypt the password field? What algo does it use to encrypt?

Reply:
Devise uses Bcrypt to securely store information. On its website it mentions that it uses “OpenBSD bcrypt() password hashing algorithm", allowing you to easily store a secure hash of your users’ passwords
ref:
https://www.freecodecamp.org/news/how-does-devise-keep-your-passwords-safe-d367f6e816eb/